### PR TITLE
Added class method to return the platformString for a specified type

### PR DESCRIPTION
**Pull request by stevel6868**

### CODE REVIEW

File: UIDeviceIdentifier/UIDeviceHardware.m [hunk 1]
```diff
@@ -22,113 +22,104 @@ + (NSString *) platform
...
-            @"iPhone11,4" : @"iPhone XS Max",
-            @"iPhone11,6" : @"iPhone XS Max",
-            @"iPhone11,8" : @"iPhone XR",
-            @"iPod1,1": @"iPod Touch 1G",
-            @"iPod2,1": @"iPod Touch 2G",
-            @"iPod3,1": @"iPod Touch 3G",
-            @"iPod4,1": @"iPod Touch 4G",
-            @"iPod5,1": @"iPod Touch 5G",
-            @"iPod7,1": @"iPod Touch 6G",
-            @"iPod9,1": @"iPod Touch 7G",
```
Comment (Coeur) [inline UIDeviceIdentifier/UIDeviceHardware.m]: Why removing iPod Touch 7G?

Comment (stevel6868) [reply UIDeviceIdentifier/UIDeviceHardware.m]: That wasn't intentional

Comment (Coeur) [reply UIDeviceIdentifier/UIDeviceHardware.m]: I mean that your whole list is missing plenty devices.
You should copy-paste it again from the current master branch.
Then squash your changes in a single commit.

**Comment by squarefrog** [comment]
Hi thanks for the PR @stevel6868 , and thanks for reviewing @Coeur . I have merged and pushed this as `1.3.0`.

**Comment by stevel6868** [comment]
Glad to contribute.  Sorry about the confusion with the missing definitions, I originally pulled via cocoa pods and for some reason it got an old version.

**Pull request merged**